library/src/main/java/com/novoda/downloadmanager/lib/db/BetterCursor.java

```diff
@@ -0,0 +1,27 @@
+package com.novoda.downloadmanager.lib.db;
+
+import android.database.Cursor;
+
+public class BetterCursor extends CursorWrapper {
+
+    public static BetterCursor wrap(Cursor cursor) {
+        return new BetterCursor(cursor);
+    }
+
+    BetterCursor(Cursor cursor) {
+        super(cursor);
+    }
+
+    public int getInt(String columnName) {
+        return getInt(getColumnIndexOrThrow(columnName));
```

Comment:
as in? Renaming it to `getXOrThrow`? Or just removing this class? I don't know if removing this class would really make the rest of the code cleaner :P